wastewater/dataModel.WasteWaterTreatment/WasteWaterTreatment-schema.json

```diff
@@ -0,0 +1,760 @@
+                    "format": "date-time"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "modifiedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "ngsildRelationship": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Relationship"
+                    ]
+                },
+                "object": {
+                    "type": [
+                        "string"
+                    ],
+                    "format": "uri"
+                },
+                "createdAt": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "modifiedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "object"
+            ]
+        },
+        "DS": {
+            "type": "object",
+            "description": "Property. Dry solids concentration measured in a tank",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "NH4": {
+            "type": "object",
+            "description": "Property. Ammonia concentration measured in a tank",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "NO3": {

```

Comment:
Subproperties defined in the NGSI LD standard are not necessary (i.e. unitcode, observedAt, etc) it should be removed